src/main/java/homework2/StringConcatenate.java

```diff
@@ -0,0 +1,34 @@
+package homework2;
+
+public class StringConcatenate {
+
+    public static String concatenate(String firstString, String secondString) {
```

Comment:
There is no need to have this as a separate method. Also, String class already has the concat() method - https://www.w3schools.com/java/ref_string_concat.asp

Reply:
thanks, done